dev/how-to/maintain/tidb-in-kubernetes/monitoring.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Please put this file to `/how-to/monitor/tidb-in-kubernetes.md`.

Reply:
Should log collecting doc been moved to the same folder as well?

Reply:
@AstroProfundis Not necessarily. Please keep it in `/how-to/maintain/tidb-in-kubernetes/`.

Reply:
Please put this file to `/how-to/monitor/tidb-in-kubernetes.md`.